pallets/xcm-app/src/lib.rs

```diff
@@ -358,6 +370,22 @@ pub mod pallet {
 
         #[pallet::call_index(3)]
         #[pallet::weight(<T as Config>::WeightInfo::register_asset())]
+        pub fn set_asset_minimum_amount(
+            origin: OriginFor<T>,
+            asset_id: AssetId,
+            min_amount: u128,
+        ) -> DispatchResultWithPostInfo {
+            let _ = T::CallOrigin::ensure_origin(origin)?;
+            let Some(multilocation) = Self::get_multilocation_from_asset_id(asset_id) else {
+                fail!(Error::<T>::MappingNotExist);
+            };
+            AssetMinimumAmount::<T>::set(multilocation, Some(min_amount));
```

Comment:
What about using of this value? Will you have another PR for that?

Reply:
you can see usage in this pr in runtime/src/trader